### ISSUE TITLE
Frames are out of sync after a while

### ISSUE THREAD

**Issue by nedasoo** [kendfrey/syncstrip#3]
Hey,
Thanks for this awesome tool. I tried making an animation for my song, the beginning fits perfectly but the more it goes into the song, the more off-sync it becomes. I checked the animation png and the y coordinates, they match nicely, so it  must be an issue with the code itself. What could cause this and how do I solve the problem?

**Comment by kendfrey** [comment]
It sounds like you may be playing the animation with the wrong frame rate. Have you tried adjusting `millisecondsPerFrame` in the json file?

**Comment by nedasoo** [comment]
Yes. I create the animation in Vegas Pro 18, set the video framerate to 50 frames per second (so that's 20ms per frame), export that as PNG images (frames) of 260x1 size (I have 260 LEDs) and then glue them into a final image. In the config I also set the millisecondsPerFrame to 20.

**Comment by nedasoo** [comment]
I have reduced the framerate to 40fps so the `millisecondsPerFrame` is now 25 and it seems to work. It seems like thats the highest framerate possible.
